extensions/giveaway-csharp-Dateline.md

```diff
@@ -0,0 +1,81 @@
+```
+
+# Installation
+In Streamer.bot in select `Import` from the top left.
+Copy the `Import Code` and paste it into the `Import String`. 
+
+## Resources
+CSV FILE:  [Resources](/assets/giveaway-csharp-Dateline/files/giveaway.csv)
+
+All code should compile, if it does not compile please reach out in discord. (Link found on Home Page)
+The following chat commands will be imported too, if you have any duplicates or wish to change them here how they are set up. 
+|Command|Action Tied to|
+|---|---|
+|`!giveaway`| 1. current status of the giveaway|
+|`!giveaway enter` |2. Joins the giveaway |
+|`!giveaway pick` |3. Pull A Winner (only Mods)|
+|`!giveaway winner` |4. posts the winner of the last giveaway |
+
+
+# Configuration
+
+This giveaway since in C# has settings in most of the actions, if any of these setting are repeated you will need to make sure that they match. All settings are changed by adjusting the Set Argument Sub-Action. Here is a run down of all the settings in each Action.
+
+Create a "Giveaway" user group -> settings -> Groups -> Add Group: Giveaway.
+
+## 1. settings Giveaway
+|Setting|Purpose|
+|---|---|
+|`startGiveaway`| This is the base number of the start date.
+|`endGiveaway`| This is the base number of the end date.
+|`requiredDays`| This is the number of at least followed days.
+|`price`| This is the (title/price).
+
+## 2. Pull a Winner
+|Setting|Purpose|
+|---|---|
+|`!giveaway pick`| This picks a random user from the .csv 
+
+## 3. Setup
+
+|Import|
+In Streamer.bot in select `Import` from the top left.
+Copy the `Import Code` and paste it into the `Import String`. 
+
+|Start Giveaway|
+you have do change the date from `YEAR, MONTH, DAY, HOURS, MINUTES, SECONDS` to (example) `2023, 01, 23, 0, 0, 0`.
+![giveway-csharp-dateline-startdate](/assets/giveaway-csharp-Dateline/images/giveway-csharp-dateline-startdate.png)
+
+|End Giveaway|
+you have do change the date from `YEAR, MONTH, DAY, HOURS, MINUTES, SECONDS` to (example) `2023, 02, 10, 14, 0, 0`.
+![giveway-csharp-dateline-enddate](/assets/giveaway-csharp-Dateline/images/giveway-csharp-dateline-enddate.png)
+
+|Price/Title|
+you have do change the price title from `Price: HERE IS MY TITLE` to (example) `Price: You can win a Waifu`.
```

Comment:
you have do change the price title from
should be
You have to change the price title from